includes/payment-methods/class-sepa-payment-gateway.php

```diff
@@ -56,16 +57,16 @@ public function payment_fields() {
 		try {
 			$display_tokenization = $this->supports( 'tokenization' ) && is_checkout();
 
-			wp_localize_script( 'wcpay-sepa-checkout', 'wcpay_config', $this->get_payment_fields_js_config() );
-			wp_enqueue_script( 'wcpay-sepa-checkout' );
+			wp_localize_script( 'wcpay-checkout', 'wcpay_config', $this->get_payment_fields_js_config() );
```

Comment:
We only register `wcpay-checkout` [script](https://github.com/Automattic/woocommerce-payments/blob/bff6eb949af53728237ad44073b52b87b40cd714/includes/class-wc-payment-gateway-wcpay.php#L515-L521) and we don't have a `wcpay-sepa-checkout` script.

Updating this to use the existing `wcpay-checkout`. Also related to this https://github.com/Automattic/woocommerce-payments/pull/1577#discussion_r614401021.